### ISSUE TITLE
[a11y]: PasswordInput with `aria-invalid` is not valid for input with role none

### ISSUE THREAD

**Issue by abbeyhrt** [carbon-design-system/carbon#11238]
This came up in slack: [thread](https://ibm-studios.slack.com/archives/C046Y0YUD/p1650036930114959). It looks like `aria-invalid` is no longer valid on an `input` with no role. Not sure if this is intended but just removing `aria-invalid` does resolve the issue, though I'm not sure if that is the right solution here. 

<img width="1792" alt="Screen Shot 2022-04-14 at 4 38 15 PM" src="https://user-images.githubusercontent.com/17281178/163625158-ce8bd482-11ec-4a91-9c88-bfab21f8aabc.png">

**Comment by samuel-jessee-ibm** [comment]
@abbeyhrt any updates on this?

**Comment by samuel-jessee-ibm** [comment]
@abbeyhrt @dakahn This issue is no longer reproducible. It seems to be fixed.
